common/retrypolicy/retry_policy.go

```diff
@@ -44,70 +43,23 @@ const (
 	// the actual failure type will be prefix + enums.TimeoutType.String()
 	// e.g. "TemporalTimeout:StartToClose" or "TemporalTimeout:Heartbeat"
 	TimeoutFailureTypePrefix = "TemporalTimeout:"
-
-	defaultInitialInterval            = time.Second
-	defaultMaximumIntervalCoefficient = 100.0
-	defaultBackoffCoefficient         = 2.0
-	defaultMaximumAttempts            = 0
-
-	initialIntervalInSecondsConfigKey   = "InitialIntervalInSeconds"
-	maximumIntervalCoefficientConfigKey = "MaximumIntervalCoefficient"
-	backoffCoefficientConfigKey         = "BackoffCoefficient"
-	maximumAttemptsConfigKey            = "MaximumAttempts"
 )
 
-func GetDefault() map[string]any {
-	return map[string]any{
-		initialIntervalInSecondsConfigKey:   int(defaultInitialInterval.Seconds()),
-		maximumIntervalCoefficientConfigKey: defaultMaximumIntervalCoefficient,
-		backoffCoefficientConfigKey:         defaultBackoffCoefficient,
-		maximumAttemptsConfigKey:            defaultMaximumAttempts,
-	}
-}
-
 // DefaultRetrySettings indicates what the "default" retry settings
 // are if it is not specified on an Activity or for any unset fields
 // if a policy is explicitly set on a workflow
 type DefaultRetrySettings struct {
-	InitialInterval            time.Duration
+	InitialInterval            time.Duration `mapstructure:"InitialIntervalInSeconds"`
```

Comment:
note the mapstructure tag to preserve the name of this key. it supports time.Duration strings as well as integer seconds though